File: Swift Day01/代码/test/test/ViewController.m
//
//  ViewController.m
//  test
//
//  Created by 1 on 16/6/14.
//  Copyright © 2016年 小码哥. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    // 1.定义不可变数组
    NSArray *array = @[@"why", @"yz"];
    // 2.定义可变数组
    NSMutableArray *arrayM = [NSMutableArray array];
    [arrayM addObject:@""];
    
    // 3.定义不可变字典
    NSDictionary *dict = @{@"name" : @"why", @"age" : @18, @"height" : @1.88};
    
    // 4.定义可变字典
    NSMutableDictionary *dictM = [NSMutableDictionary dictionary];
    [dictM setObject:@"" forKey:@""];
    
    dictM[@"age"] = @18;
    
    dictM objectForKey:<#(nonnull id)#>
}

@end

File: swift day3- 语法高级/代码/12-常见注释/12-常见注释/ViewController.swift
//
//  ViewController.swift
//  12-常见注释
//
//  Created by 1 on 16/6/17.
//  Copyright © 2016年 小码哥. All rights reserved.
//

import UIKit
/*
 1.单行注释 //
 2.多行注释 /* */ 在swift中多行注释可以嵌套使用
 3.文档注释 ///
 4.分组注释 // MARK:- 分组注释
*/

class ViewController: UIViewController {
    
    // MARK:- 所有的属性
    /// 姓名
    var name : String = ""
    
    
    // MARK:- 所有的方法
    // MARK: 控制器的View加载完成
    // viewDidLoad方法
    override func viewDidLoad() {
        super.viewDidLoad()
        
        test()
    }
    
    /// 测试方法
    func test() {
        
    }
}

